src/main/java/baritone/process/MineProcess.java

```diff
@@ -103,8 +104,9 @@ public PathingCommand onTick(boolean calcFailed, boolean isSafeToCancel) {
                 return null;
             }
         }
-        if (!Baritone.settings().allowBreak.value) {
-            logDirect("Unable to mine when allowBreak is false!");
+//        baritone.getPathingBehavior().updateIsCloseToHome();
+        if (!Baritone.settings().allowBreak.value || baritone.getPathingBehavior().isCloseToHome()) {
+            logDirect("Unable to mine when allowBreak is false or when you\'re too close to home!");
```

Comment:
If you do the checks in the CalculationContexts per block, then you should remove this check and replace it by blacklisting any blocks near a home. If you decide to cache whether the player is near a home for pathing, then keep this. Just so it's consistent.